RELEASE-NOTES.md

```diff
@@ -1,3 +1,33 @@
+### 0.11.0
+- Added virtual machine with automating configuration of golem and concent for testing purpose (#146, #147, #148, #182, #186).
+- Added liveness and readiness probe to geth (#44).
+- Added support for enabling routing based on golem-messages version on a single cluster (#289).
+- Added ability to manage resource limits for applications on different clusters (#257).
+- Added ability to switching between an internal and external geth instance (#285).
+- Added ability to generate a key pair for signing-service (#335).
+- Deployment process is now execute in home user path on the concent-builder server (#223).
+- Bugfix: added missing `gcloud` and `kubectl` packages source (#275).
+- Bugfix: added missing upload limit for proxy nginx router (#320).
+- Bugfix: upgraded `rustc` package from debian testing repository after golem requires version that isn't available on debian stable repository (#307, #328).
+- Bugfix: `python3.6` now is installed from pyenv source after debian deleted it from testing repository (#321).
+- Bugfix: deleted duplicate directive in the signing-service docker image (#337).
```

Comment:
This is so trivial that I would treat it as overall cleanup and not mention it in the changelog.